### DISCUSSION TITLE
Dependabot - How to determine which packages are managed by NX

### DISCUSSION THREAD

**Discussion by jvpelt** [nrwl/nx#18307] [Show and Tell]
Wondering how other people are using dependabot.

I'm now manually keeping track of the non NX added package names in the "allow" list of my dependabot.yml. Is there a way to automatically determine which packages in the dependencies/devDependencies are added/managed by NX?

**Comment by janeklb** [comment]
+1 I'm also interested in getting this kind of information from NX (very similar usecase to @jvpelt)

**Comment by liam-daley** [comment]
+1

**Comment by lwensveen** [comment]
Yeah this really has been grinding my gears, even manually keeping track of this is a major pain.
